### PR TITLE
Update GLTFSceneExporter.cs

### PR DESCRIPTION
**Pull request by kknsy**
revise SaveGLB

### CODE REVIEW

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ private struct ImageInfo
 		private List<ImageInfo> _imageInfos;
 		private List<Texture> _textures;
 		private List<Material> _materials;
+		private bool _imagesInInternalBuffer;
```
Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs]: replace spaces with tabs

Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs]: bools should be named a conditional, like "_shouldUseInternalBufferForImages"

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs [hunk 2]
```diff
@@ -1113,7 +1126,73 @@ private ImageId ExportImage(Texture texture, TextureMapType texturMapType)
...
+				break;
+			    case TextureMapType.Bump:
+				Graphics.Blit(texture, destRenderTexture, _normalChannelMaterial);
+				break;
+			    default:
+				Graphics.Blit(texture, destRenderTexture);
+				break;
+			}
+
+			//Graphics.Blit(texture, destRenderTexture);
```
Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs]: comment or remove

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs [hunk 3]
```diff
@@ -1113,7 +1126,73 @@ private ImageId ExportImage(Texture texture, TextureMapType texturMapType)
...
+		    }
 
+		    var image = new GLTFImage();
+		    image.MimeType = "image/png";
+
+		    var byteOffset = _bufferWriter.BaseStream.Position;
+
+		    {//
+			var destRenderTexture = RenderTexture.GetTemporary(texture.width, texture.height, 24, RenderTextureFormat.ARGB32, RenderTextureReadWrite.sRGB);
+			GL.sRGBWrite = true;
```
Comment (blgrossMS) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs]: this is only true for basecolor/albedo and emissive.

Comment (kknsy) [reply UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs]: https://github.com/KhronosGroup/UnityGLTF/issues/186  This is the original author，I just made a simple modification. I don’t understand the actual meaning. I don’t know how to change it.

**Pull request merged**